### PR TITLE
Add grpc node config

### PR DESCRIPTION
**Pull request by aoudiamoncef**
https://massa--143.org.readthedocs.build/en/143/technical-doc/grpc.html

### CODE REVIEW

File: technical-doc/grpc.rst [hunk 1]
```diff
@@ -22,6 +22,21 @@ Find the complete Massa `gRPC <https://grpc.io/>`_ specification `here
 
     Massa gRPC support is experimental. This API is not yet stable and may change in the future.
 
+To enable WebSocket support in Massa node, edit file ``massa-node/config/config.toml`` (create it if absent) with the
```
Comment (AurelienFT) [inline technical-doc/grpc.rst:25]:
```suggestion
To enable gRPC support in Massa node, edit file ``massa-node/config/config.toml`` (create it if absent) with the
```

**Review by AurelienFT — APPROVED** [review]

**Comment by qdrn** [comment]
@aoudiamoncef can you make a separate PR to implement the corrections I've made ?

**Comment by aoudiamoncef** [comment]
@qdrn sorry, what corrections ? 

We have an ongoing PR https://github.com/massalabs/massa-docs/pull/145, feel free to make improvements before merging

**Comment by qdrn** [comment]
> @qdrn sorry, what corrections ?
> 
> We have an ongoing PR #145, feel free to make improvements before merging

Just above.

**Pull request merged**